antarest/core/exceptions.py

```diff
@@ -8,6 +8,53 @@ class ShouldNotHappenException(Exception):
     pass
 
 
+class STStorageFieldsNotFoundError(HTTPException):
+    """Fields of the short-term storage are not found"""
+
+    def __init__(self, study_id: str, area_id: str, storage_id: str) -> None:
+        detail = (
+            f"Error in the study '{study_id}',"
+            f" the short-term storage configuration of area '{area_id}' is invalid:"
+            f" fields of storage '{storage_id}' not found"
+        )
```

Comment:
Done

Reply:
I think API users should have a clear message about the error and it's best to give as much context as possible. For me, the study ID, zone ID and storage ID are important because one of the three is potentially wrong.